### PR TITLE
:bug: Fixes crash when solution file in Fettle config has no folder in it and doesn't start with ".\"

### PR DESCRIPTION
**Pull request by sanderrorije**

### CODE REVIEW

**Comment by oliwennell** [comment]
Thanks @sanderrorije ! I'll take a look soon

File: src/Core/Config.cs [hunk 1]
```diff
@@ -46,6 +47,13 @@ public Config WithPathsRelativeTo(string baseDirectory)
 
         public bool HasCustomTestRunnerCommand => !string.IsNullOrEmpty(CustomTestRunnerCommand);
 
+	    public string GetSolutionFolder()
```
Comment (oliwennell) [inline src/Core/Config.cs]: I like how you extracted this into a method here 👍

Comment (sanderrorije) [reply src/Core/Config.cs]: Thanks!

Comment (oliwennell) [reply src/Core/Config.cs]: No problem, that's why we have code reviews :)

No more action required from you, I'll merge the PR and make a new Fettle release soon (hopefully tomorrow).

Thanks for contributing 👍

Comment (sanderrorije) [reply src/Core/Config.cs]: Ok thanks. This was my first contribution to a GitHub project, and I liked it. Perhaps I can contribute more in the future!

**Comment by oliwennell** [comment]
Fixes #64

**Pull request merged**